branchpro/_dataset_library_api.py

```diff
@@ -0,0 +1,29 @@
+#
+# DatasetLibraryAPI Class
+#
+# This file is part of BRANCHPRO
+# (https://github.com/SABS-R3-Epidemiology/branchpro.git) which is released
+# under the BSD 3-clause license. See accompanying LICENSE.md for copyright
+# notice and full license details.
+#
+
+import os
+import pandas as pd
+
+
+class DatasetLibrary:
+    """DatasetLibrary Class:
+    Class for reading the files in the data library. These files are in the
+    .csv format.
+    """
+    def __init__(self):
+        self._directory = os.path.join(
+            os.path.dirname(__file__), 'data_library')
+
+    def french_flu(self):
+        """
+        Reads the datafile on the weekly number of flue incidences registered
+        in France between 1984 (week 44) and 2020 (week 42).
+        """
```

Comment:
Here the docstring could give a little more away about the data. I like the first sentence, but maybe we can say which columns can be expected in the dataframe? E.g.:
```
def french_flu(self):
        """
        Returns a dataset on the weekly incidence numbers of the annual flue
        outbreak in France between the years 1984 and 2020.
       
       The data is returned as a :class:`pandas.DataFrame` with columns (....). (Potentially explain
       explain columns if not self-explanatory).
        """
```